packages/@jsii/dotnet-analyzers/test.sh

```diff
@@ -1,6 +0,0 @@
-#!/bin/bash
-set -euo pipefail
-
-# Run unit tests
-echo "Running library unit tests"
-dotnet test -c Release ./src/Amazon.JSII.Analyzers.UnitTests
```

Comment:
Verify that these unit tests are still executed.

Reply:
They definitely are :) I've checked the logs and all.